frontend/src/pages/MilestonesEditor.module.scss

```diff
@@ -146,7 +147,6 @@
 .topbar {
   @extend .layout-row;
   min-height: 60px;
-  overflow-x: auto;
```

Comment:
This sadly brings back the double-scrollbar on the bottom of the page. 
With the line, the menu content would not pop over elements (instead locating itself inside the top bar, in a vertical scrollbox)

Reply:
Well, I guess we can live with that. The double-scroll bar becomes visible only when your screen with is under 1035px, so it shouldn't be there too often and even when it's there, it's nothing too bad. If you'd like to fix it, you could for example make the navbar go on two lines (if possible) or hide certain not-so-important-things there, such as the ones in the image, when going under certain width thresholds. Or make the weighted shares bar narrower etc. But IMO it's okay to leave as it is as well

![image](https://user-images.githubusercontent.com/80753884/176730488-1a4cbff6-6dc6-4a07-8488-b0d4865291bb.png)

Reply:
Good suggestions, but I'll leave it as is due to the time constraints